packages/slycat/web/server/engine.py

```diff
@@ -90,6 +90,7 @@ def abspath(path):
   dispatcher.connect("get-configuration-parsers", "/configuration/parsers", slycat.web.server.handlers.get_configuration_parsers, conditions={"method" : ["GET"]})
   dispatcher.connect("get-configuration-remote-hosts", "/configuration/remote-hosts", slycat.web.server.handlers.get_configuration_remote_hosts, conditions={"method" : ["GET"]})
   dispatcher.connect("get-configuration-smb-remote-hosts", "/configuration/smb-remote-hosts", slycat.web.server.handlers.get_configuration_smb_remote_hosts, conditions={"method" : ["GET"]})
+  dispatcher.connect("get-configuration-smb-domains", "/configuration/smb-domains", slycat.web.server.handlers.get_configuration_smb_domains, conditions={"method" : ["GET"]})
```

Comment:
add a doc page for the new endpoint

Reply:
See d528cda6f9ccc4d260a0526fb62e74026a9b9fba